docs/UserGuide.md

```diff
@@ -62,111 +74,128 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
 **:information_source: Notes about the command format:**<br>
 
 - All command follow this format:<br>
```

Comment:
might be good to mention that all identifiers are case sensitive

Reply:
Have added this in